ext/fs/std_fs.rs

```diff
@@ -1021,20 +1021,7 @@ fn exists(path: &Path) -> bool {
 
   #[cfg(windows)]
   {
-    use std::os::windows::ffi::OsStrExt;
-
-    use winapi::um::fileapi::GetFileAttributesW;
-    use winapi::um::fileapi::INVALID_FILE_ATTRIBUTES;
-
-    let path = path
-      .as_os_str()
-      .encode_wide()
-      .chain(std::iter::once(0))
-      .collect::<Vec<_>>();
-    // Safety: `path` is a null-terminated string
-    let attrs = unsafe { GetFileAttributesW(path.as_ptr()) };
-
-    attrs != INVALID_FILE_ATTRIBUTES
```

Comment:
This is fishy 👀 can you explain why you opted to remove this low-level API call?

Reply:
Looks like this was done for perf: https://github.com/denoland/deno/pull/24613 -- but also before we were doing way too much work in our stat calls.

Overall this seems fine.